### PR TITLE
Video tutorial for youtube administrator

### LINKED ISSUE #5217: Add recorded video tutorial for YouTube administrators
Create a video walkthrough of the youtube playlist clean up for current and future Youtube administrators.

### PR DESCRIPTION
**Pull request by markyjackson-taulia**
Signed-off-by: markyjackson-taulia <marky.r.jackson@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #5217

### CODE REVIEW

**Comment by castrojo** [comment]
Are we sure we want this on gdrive or do we want it on youtube in an admin playlist or something?

**Comment by markyjackson-taulia** [comment]
> Are we sure we want this on gdrive or do we want it on youtube in an admin playlist or something?

I wasn't aware there was such a playlist. I am happy to move it there.

**Comment by castrojo** [comment]
I don't think we have one, I'm just thinking if in the future we have videos of ourselves showing ourselves how we manage our stuff it might be useful to have those on a playlist and accessible via youtube.  

"Hi I'm a new cloud native project how should I manage youtube?" 
"This is how we do it in k8s, go to this playlist and watch these videos." 

^^^ that might be useful to other projects moving forward, basically we could create a meta-playlist for things like that.

**Comment by markyjackson-taulia** [comment]
Should I create and `Admin` playlist and upload the video there?

File: communication/youtube/youtube-guidelines.md [hunk 1]
```diff
@@ -189,6 +189,9 @@ Below are a good and incorrect example:
 * Incorrect description: `Data Protection WG Bi-Weekly Meeting for 2/26/2020`
 * Correct description: `Data Protection WG Bi-Weekly Meeting for 20200226`
 
+A short video tutorial of the editing of descriptions and playlists can be found here:
+* https://drive.google.com/file/d/1_GwcKgnEhVlnm7w2HJKS3zIybXq5Bavj/view?usp=sharing
```
Comment (mrbobbytables) [inline communication/youtube/youtube-guidelines.md:193]: We should probably put this up in YT 🤣 
what do you think?

Comment (markyjackson-taulia) [reply communication/youtube/youtube-guidelines.md:193]: +1 what playlist should this be under? @castrojo should we create an admins playlist?

**Comment by markyjackson-taulia** [comment]
@castrojo & @mrbobbytables 
Should I place this video in the contribex playlist or create a new admins playlist?

**Comment by mrbobbytables** [comment]
I wouldn't worry about adding it to a playlist, it can just live in the channel.

**Comment by markyjackson-taulia** [comment]
@castrojo @mrbobbytables Video has been uploaded to youtube (not in any playlist) and doc has been updated to reflect the youtube video link

**Comment by castrojo** [comment]
Saw it, nice work!

**Comment by castrojo** [comment]
/lgtm

Oops. :D

**Comment by chris-short** [comment]
/lgtm

/approve

¯\\\_(ツ)\_/¯

**Comment by chris-short** [comment]
/assign @castrojo 

@k8s-ci-robot told me to!

**Pull request merged**